src/json_validate.py

```diff
@@ -112,15 +112,15 @@ def validate_trial_generate_json(trial_generate_json):
             validate(instance=trial_generate_json, schema=delete_experiment_schema, format_checker=draft7_format_checker)
         elif not str(trial_generate_json["operation"]) or not str(trial_generate_json["operation"]).strip() or \
                 trial_generate_json["operation"] is None:
-            errorMsg = "Parameters" + HPOErrorConstants.VALUE_MISSING
+            errorMsg = "Parameters" + "operation" + HPOErrorConstants.VALUE_MISSING
         else:
             errorMsg = HPOErrorConstants.INVALID_OPERATION
 
         return errorMsg
     except jsonschema.exceptions.ValidationError as err:
         # Check if the exception is due to empty or null required parameters and prepare the response accordingly
         if any(word in err.message for word in HPOErrorConstants.JSON_NULL_VALUES):
-            errorMsg = "Parameters" + HPOErrorConstants.VALUE_MISSING
+            errorMsg = "Parameters" + word + HPOErrorConstants.VALUE_MISSING
```

Comment:
Changes in line 123, in validate_trial_generate_json results in the following error:
    errorMsg = "Parameters" + word + HPOErrorConstants.VALUE_MISSING
NameError: name 'word' is not defined